solidago/tests/test_mehestan_primitives.py

```diff
@@ -1,7 +1,8 @@
+from unittest import TestCase
```

Comment:
It would make sense to use pytest-style in Solidago, yes. No objection to migrate the tests in this file.

In the backend itself, the use of test classes is mostly motivated by the tools provided by Django to interact with the test database (wrap tests in transactions, flush the db, etc.). That's the documented way:
https://docs.djangoproject.com/en/4.2/topics/testing/overview/#writing-tests